Core/include/Acts/TrackFinding/MeasurementSelector.hpp

```diff
@@ -233,12 +233,12 @@ class MeasurementSelector {
 
   double calculateChi2(
       double* fullCalibrated, double* fullCalibratedCovariance,
-      TrackStateTraits<MultiTrajectoryTraits::MeasurementSizeMax,
-                       false>::Parameters predicted,
-      TrackStateTraits<MultiTrajectoryTraits::MeasurementSizeMax,
-                       false>::Covariance predictedCovariance,
-      TrackStateTraits<MultiTrajectoryTraits::MeasurementSizeMax,
-                       false>::Projector projector,
+      const TrackStateTraits<MultiTrajectoryTraits::MeasurementSizeMax,
```

Comment:
Just to note: these are map types, that have reference semantics and only store a pointer to the backing store. So creating const refs here simply means the compiler creates a temporary that it hangs on to. 

Did you observe performance improvements with this?

Reply:
still evaluating. But there are for sure other things I'd like to change. So if this specific change makes no difference I will revert it